src/array.jl

```diff
@@ -580,13 +580,6 @@ Return levels which appear in `A`, in the same order as [`levels`](@ref)
 """
 function unique end
 
-"""
```

Comment:
OK, sorry, I hadn't spotted the difference between `droplevels!` and `droplevels`. That's just a typo: please just add the `!` in `function droplevels end` (the `droplevels` function isn't supposed to exist and has no methods currently).